features/checkout/preventing_cart_to_be_modified_after_checkout.feature

```diff
@@ -0,0 +1,71 @@
+@checkout
+Feature: Preventing cart to be modified after checkout by Customer
+    In order to not modify cart data after checkout
+    As a Customer
+    I want my cart to blocked from making changes after checkout
```

Comment:
```suggestion
Feature: Preventing cart to be modified after checkout
    In order to have order immutable after checkout
    As a Customer
    I want to not be able to modify placed order
```
WDYT?

Reply:
But `I want not to be able to modify placed order`